src/Sylius/Component/Core/spec/Promotion/Checker/NthOrderRuleCheckerSpec.php

```diff
@@ -92,4 +98,18 @@ function it_should_recognize_subject_as_not_eligible_if_nth_order_is_equal_with_
 
         $this->isEligible($subject, ['nth' => 10])->shouldReturn(true);
     }
+
+    function it_recognizes_subject_as_not_eligible_if_configuration_is_invalid(OrderInterface $subject)
+    {
+        $this->isEligible($subject, [])->shouldReturn(false);
+        $this->isEligible($subject, ['nth' => '10'])->shouldReturn(false);
```

Comment:
I would change `'nth' => '10'` to `'nth' => 'string'` to highlight it is a matter of invalid type.